packages/config/src/layer2s/mode.ts

```diff
@@ -25,4 +35,52 @@ export const mode: Layer2 = upcoming({
       ],
     },
   },
+  upgradeability,
+  l1StandardBridgeEscrow: EthereumAddress(
+    '0x735aDBbE72226BD52e818E7181953f42E3b0FF21',
+  ),
+  rpcUrl: 'https://mainnet.mode.network/',
+  sequencerAddress: EthereumAddress(
+    discovery.getContractValue('SystemConfig', 'batcherHash'),
+  ),
+  inboxAddress: EthereumAddress('0x24e59d9d3bd73ccc28dc54062af7ef7bff58bd67'),
+  genesisTimestamp: new UnixTime(1700125343),
+
+  l2OutputOracle: discovery.getContract('L2OutputOracle'),
+  portal: discovery.getContract('OptimismPortal'),
+  //stateDerivation: DERIVATION.OPSTACK('MODE'),
+  isNodeAvailable: true,
```

Comment:
```suggestion
  isNodeAvailable: 'UnderReview',
```

it's available only if we can add the state derivation section to it